cmake/ConfigureCompiler.cmake

```diff
@@ -44,6 +45,14 @@ else()
   add_definitions(-DUSE_UCONTEXT)
 endif()
 
+if(USE_CCACHE)
+	FIND_PROGRAM(CCACHE_FOUND "ccache")
```

Comment:
If turned off by default, I think cmake should fail if it can't find ccache. Otherwise it might be confusing to a user who might think she has ccache installed but cmake can't find it and silently compiles without using it.

Reply:
By default, I believe that ccache stores its files in a hidden directory within the user home directory. Doing this without the user consent could cause problems.

Reply:
I am fine with not turning on cchache by default. But I think the current implementation is not very user-friendly as it is a bit confusing.

If I call `cmake -DUSE_CCACHE=ON $fdb_src` and cmake finishes without an error, I expect it to compile using `ccache`. However, this is not what would happen here. This is why I suggest to throw an error there.